docs/diagrams/testing-component-diagram.puml

```diff
@@ -0,0 +1,46 @@
+@startuml
+
+package "Rust Common" {
+    [Orderbook]
+    [Execute]
+    [Rust Lightning]
+}
+
+node "App" {
```

Comment:
Would be great if we could align on a naming - in https://github.com/get10101/10101/pull/31 we use a different terminology.

```
App = mobile
FRB / App Backend = native
```

Reply:
I think `mobile` and `native` are good in context of the app, but we have to be clear that this is in context of the app and not the coordinator.